Controller/NotifyHandler.php

```diff
@@ -196,21 +196,18 @@ private function _onNotificationSuccess($oResponse, $oBackendService, $sPaymentI
         }
 
         $oOrder = oxNew(Order::class);
-        $bSavedTransaction = self::_saveIfUnmatchedPoiPiaTransaction(
+        if (!self::_handleUnmatchedTransactions(
             $sTransactionId,
             $oResponse,
             $oBackendService,
             $oOrder
-        );
-
-        if (!$bSavedTransaction) {
+        )) {
             ResponseHandler::onSuccessResponse($oResponse, $oBackendService, $oOrder);
         }
-        return;
     }
 
     /**
-     * Saves POI/PIA transaction if it is unmatched, and returns true. Otherwise returns false.
+     * Returns true if no matching order was found for transaction. Otherwise returns false.
```

Comment:
Please change this description to reflect the method name: "Handles transactions that cannot be matched to an existing order. Returns true if handled, false otherwise."

Reply:
Done.